src/compose.ts

```diff
@@ -46,12 +46,16 @@ services:
  *
  * @param c The input composition as a plain JS object
  */
-export function normalize(c: any): Composition {
-	if (!_.isObject(c)) {
+export function normalize(o: any): Composition {
+	if (!_.isObject(o)) {
 		throw new ValidationError('Invalid composition format');
 	}
 
 	let version: SchemaVersion;
+	let c = o as {
```

Comment:
I'm not completely sure why this is necessary (on my laptop it builds fine without any of this) so I guess it could have to do with a different TypeScript version or config being used on Circle for some reason?

Reply:
It could be Lodash types — some recent version had a bug — I don’t remember the details though and the workaround is fine anyway.